config/template/stop-extrabbit

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+{% include "proc-stub" %}
+
+echo "stopProcess :: external-rabbit"
```

Comment:
RabbitMQ is something that CBS wants to control.  So it wants a start/stop script.  However, RabbitMQ is a component controlled by something else (i.e. internal deployment tools).  To avoid them fighting over control, we have mock commands for CBS.